pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -0,0 +1,166 @@
+      <Versions>8</Versions>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="fedora">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>23;24;25;26</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="gentoo">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="linuxmint.17">
+      <Parent>ubuntu.14.04</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>1;2;3</Versions>
+    </RuntimeGroup>
+    <RuntimeGroup Include="linuxmint.18">
+      <Parent>ubuntu.16.04</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>1</Versions>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="ol">
+      <Parent>rhel</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>7;7.0;7.1;7.2</Versions>
+      <ApplyVersionsToParent>true</ApplyVersionsToParent>
+    </RuntimeGroup>
+
+    <!-- all opensuse releases are independent -->
+    <RuntimeGroup Include="opensuse">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>13.2;42.1</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="osx">
+      <Parent>unix</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>10.10;10.11;10.12</Versions>
+    </RuntimeGroup>
+
+    <!-- rhel 6 is independent -->
+    <RuntimeGroup Include="rhel">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>6</Versions>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="rhel">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4</Versions>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="tizen">
+      <Parent>linux</Parent>
+      <Architectures>armel</Architectures>
+      <Versions>4.0.0</Versions>
+    </RuntimeGroup>
+
+    <!-- ubuntu releases are compatible on major versions-->
+    <RuntimeGroup Include="ubuntu">
+      <Parent>debian</Parent>
+      <Architectures>x64;x86;arm</Architectures>
+      <Versions>14.04;14.10;15.04;15.10</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
+    </RuntimeGroup>
+    <RuntimeGroup Include="ubuntu">
+      <Parent>debian</Parent>
+      <Architectures>x64;x86;arm;arm64</Architectures>
+      <Versions>16.04;16.10</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
+    </RuntimeGroup>
+
+    <_winGroup Include="win">
+      <Parent>any</Parent>
+      <OmitVersionDelimiter>true</OmitVersionDelimiter>
+      <AdditionalQualifiers>aot</AdditionalQualifiers>
+    </_winGroup>
+    <RuntimeGroup Include="@(_winGroup)">
+      <Architectures>x64;x86;arm;arm64</Architectures>
+      <Versions>7;8;81;10</Versions>
+    </RuntimeGroup>
+    <!--RuntimeGroup Include="@(_winGroup)">
```

Comment:
What's the plan for these RuntimeGroups?  Are we going to include them?  or keep them commented out?

Reply:
Remove them.  This technique was actually causing a problem since it would result in win8-arm importing win-arm, and then win, and actually brought it into the graph higher than win7.  This is what I meant by using OmitRIDs being dangerous.